File: LeetCode/2810. Faulty Keyboard/answer.py
# 2810. Faulty Keyboard
# https://leetcode.com/problems/faulty-keyboard/

class Solution:
    def finalString(self, s: str) -> str:
        result = ""

        for c in s:
            if c == "i":
                result = result[::-1]
            else:
                result += c

        return result

File: LeetCode/2796. Repeat String/answer.ts
// 2796. Repeat String
// https://leetcode.com/problems/repeat-string/

declare global {
  interface String {
    replicate(times: number): string;
  }
}

String.prototype.replicate = function (times: number) {
  let result = '';
  for (let i = 0; i < times; i++) {
    result += this;
  }

  return result;
};
